llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -18143,53 +18143,63 @@ performActiveLaneMaskCombine(SDNode *N, TargetLowering::DAGCombinerInfo &DCI,
       (!ST->hasSVE2p1() && !(ST->hasSME2() && ST->isStreaming())))
     return SDValue();
 
-  if (!N->hasNUsesOfValue(2, 0))
+  unsigned NumUses = N->use_size();
+  unsigned MaskMinElts = N->getValueType(0).getVectorMinNumElements();
```

Comment:
Please can you try to reduce the usage of getVectorMinNumElements()? In this instance you can use `isKnownMultipleOf()` and `divideCoefficientBy()` so the code is not, or at least not as, specific to scalable vectors.